<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterM</class>
 <widget class="QDialog" name="AjouterM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#1b1a1c;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:22px;
color:#fff;</string>
   </property>
   <property name="text">
    <string>INSCRIPTION NOUVEAU MEMBERE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
  border: 2px solid #ccc;
  border-radius: 5px;
  font-size: 16px;
  background-color:#fff;

  width: 100%;
color:black;</string>
   </property>
   <property name="placeholderText">
    <string>prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>Nom de famille</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>NUMERO DE TELEPHONE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>ANNE DE NAISSANCE(YYYY)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
  border: 2px solid #ccc;
  border-radius: 5px;
  font-size: 16px;
  background-color:#fff;

  width: 100%;
color:black;</string>
   </property>
   <property name="placeholderText">
    <string>Numéro de télé</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
  border: 2px solid #ccc;
  border-radius: 5px;
  font-size: 16px;
  background-color:#fff;

  width: 100%;
color:black;</string>
   </property>
   <property name="placeholderText">
    <string>date de naissance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
  border: 2px solid #ccc;
  border-radius: 5px;
  font-size: 16px;
  background-color:#fff;

  width: 100%;
color:black;</string>
   </property>
   <property name="placeholderText">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>HOMME</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>FEMME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>SPORT CHOISI</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>290</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>ABONNEMET (MOIS)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>360</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>MONTANT PAVE (AVANCE)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
  border: 2px solid #ccc;
  border-radius: 5px;
  font-size: 16px;
  background-color:#fff;

  width: 100%;color:black;</string>
   </property>
   <property name="placeholderText">
    <string>montant d'avance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>450</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>DATE DE DEBUT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>520</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>A PAYER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>520</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>150</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>520</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>DH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>540</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>RESTANT A PAYER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>540</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>RESTANT A PAYER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>540</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="text">
    <string>RESTANT A PAYER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>630</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#b6393a;
color:#fff;
border-radius:12px;
font-size:22px;</string>
   </property>
   <property name="text">
    <string>VALIDER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>630</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#cc7200;
color:#fff;
border-radius:12px;
font-size:22px;</string>
   </property>
   <property name="text">
    <string>ANNULER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#dd4b3a;
color:#fff;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>240</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fff;</string>
   </property>
   <item>
    <property name="text">
     <string>GYM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TEKWONDO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KARATE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GEDO</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>320</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fff;</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>470</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fff;  border: 2px solid #ccc;
  border-radius: 5px;
  font-size: 16px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
